library/objective-c/EnvoyEngine.h

```diff
@@ -218,6 +218,15 @@ extern const int kEnvoyFilterResumeStatusResumeIteration;
 
 @end
 
+#pragma mark - EnvoyNativeFilterConfig
+
+@interface EnvoyNativeFilterConfig : NSObject
+
+@property (nonatomic, strong) NSString *name;
+@property (nonatomic, strong) NSString *typedConfig;
```

Comment:
docs for these, especially since they're just strings?

Reply:
doesn't seem like we traditionally add docstrings here? I added them in the EngineBuilder method.

Reply:
ok if it's in the builder I guess that's sufficient - the reason i called it out here is because it's quite unclear to the reader what a `typedConfig` should actually contain